functions/stdlib/quick_exit.c

```diff
@@ -0,0 +1,39 @@
+/* quick_exit( int )
+
+   This file is part of the Public Domain C Library (PDCLib).
+   Permission is granted to use, modify, and / or redistribute at will.
+*/
+
+#include <stdlib.h>
+
+#ifndef REGTEST
+
+/* TODO - "...except that a function is called after any previously registered
+   functions that had already been called at the time it was registered."
```

Comment:
Context for this quote would help

Reply:
It's from the C standard (copied from exit.c), from the section where `quick_exit` and `exit` are explained. I added a small note.

Reply:
What is there left `TODO` then? Is this about the case where functions are registered from within the callbacks? I'm still not sure what is meant by that comment.

Reply:
If I may? I added that comment to my implementation of exit.c to remind myself that I am not 100% sure I understood the meaning of the standard here, and that I want (TODO) to go back and double-check I did not miss something subtle.

The comment just got copied to quick_exit.c, as the implementations are virtually identical.